tensorflow/lite/delegates/xnnpack/binary_elementwise_tester.cc

```diff
@@ -66,7 +66,7 @@ void BinaryElementwiseTester::Test(tflite::BuiltinOperator binary_op,
   if (Input1Static()) {
     ASSERT_FALSE(Input2Static());
   }
-  if (FP16Weights() || INT8Weights()) {
+  if (FP16Weights() || INT8Weights() || INT8ChannelWiseWeights()) {
     ASSERT_TRUE(Input1Static() || Input2Static());
   }
 
```

Comment:
Check that if channelwise weights are used, the static input has at least one dimension (i.e. isn't a scalar)

Reply:
Done

Reply:
Should be `if (INT8ChannelWiseWeights() && Input1Static())` ... + the same for input2

Reply:
Sorry, my bad. Fixed